skipper.go

```diff
@@ -66,6 +53,10 @@ type Options struct {
 	// Skip TLS certificate check for etcd connections.
 	EtcdInsecure bool
 
+	// Kubernetes API base address for ingress. If not specififed,
+	// Kubernetes ingress is disabled.
+	KubernetesAPIAddress string
```

Comment:
i follow this naming convention to use address when it is only an address of a server, like http://localhost:8080 but no path or else. The innkeeper one just may be a full url, i don't remember. Is this dumb? Or are there better conventions for this distinction?